src/testers.jl

```diff
@@ -100,6 +100,21 @@ function _make_jvp_call(fdm, f, xs, ẋs, ignores)
     return jvp(fdm, f2, sigargs...)
 end
 
+"""
+    _basis_vectors(x::T) -> Vector{T}
+
+Get a set of basis (co)tangent vectors for `x`.
+
+This function assumes that the (co)tangent vectors are of the same type as `x` and requires
```

Comment:
It'd be nice to give basis vectors of the same type as the result of `rand_tangent` instead, but I'm not certain how to do that.